earthfile2llb/flags.go

```diff
@@ -0,0 +1,22 @@
+package earthfile2llb
+
+import (
+	flags "github.com/jessevdk/go-flags"
+)
+
+type runOpts struct {
+	Push            bool     `long:"push" description:"Execute this command only if the build succeeds and also if earthly is invoked in push mode"`
+	Privileged      bool     `short:"P" long:"privileged" description:"Enable privileged mode"`
```

Comment:
The short version would not correspond to the Dockerfile equivalent. Although I can see how this matches to the short version of `--allow-privileged`.

I'm starting to think that the long version of flags in scripts is better because it helps with readability. Giving the user two options means that there will be a mixture of Earthfiles out there some using long version, some using short version and some using a mix, which makes Earthfiles less readable overall.

What do you think of not providing a short version of any flag in Earthfiles (even `--allow-privileged`)? Just long versions.

Reply:
I'm honestly fine with that, `--allow-privileged` really stands out to a user reading the Earthfile, where as `-P` might not jump out as much as a security concern.

We could still make this switch and just keep the long version, then possibly come back to it if we get a lot of requests for a shorter version. The only real downside is users (myself included) would have to learn how to consistently spell privileged correctly.

Reply:
Sounds like a plan. I agree that the library switch is worthwhile nevertheless.